lib/datadog/tracing/contrib/mongodb/integration.rb

```diff
@@ -17,6 +17,9 @@ class Integration
 
           # @public_api Changing the integration name or integration options can cause breaking changes
           register_as :mongo, auto_patch: true
+          def self.gem_name
+            'mongo'
+          end
 
           def self.version
             Gem.loaded_specs['mongo'] && Gem.loaded_specs['mongo'].version
```

Comment:
Minor: Maybe not in scope for this PR, but it seems to me that now that we have a `gem_name`, we could probably implement a default `version` that's just `Gem.loaded_specs[gem_name]&.version` since that seems to cover (almost?) all of them :)

Reply:
Would the version returned by `Gem.loaded_specs[gem_name]&.version` be a version that is being tested or just a version that is loaded in the environment? If it is pulling the version from the test suite that'd be fine and could be a good improvement for a future PR.

Reply:
If I understand your question correctly, `Gem.loaded_specs[gem_name]` is the version that's either loaded OR available to load (it may have not been `require`d yet -- which is why we usually have a separate `loaded?` check for that).

This `version` is usually combined with the `MINIMUM_VERSION` constant to determine the outcome for `compatible?`.

Reply:
Yeah that answers it thanks (I think 😄)!

The goal that we have with documentation for "supported" integrations cross language is to report the minimum and maximum _tested_ versions only. So I don't think we can safely use this value.

To elaborate for this instance of `mongo`:

We have `MINIMUM_VERSION = Gem::Version.new('2.1.0')`.
If in the integration tests for `mongo` the minimum version tested is `2.5.0` we will report `2.5.0` as the minimum version _supported_ moving forward.